basyx.submodelrepository/basyx.submodelrepository.component/src/main/resources/application.properties

```diff
@@ -3,12 +3,13 @@ server.port=8081
 spring.application.name=Submodel Repository
 basyx.backend = InMemory
 
-# basyx.backend = MongoDB
-#spring.data.mongodb.host=localhost
+#basyx.backend = MongoDB
+#spring.data.mongodb.host=mongo
+# or spring.data.mongodb.host=127.0.0.1
 #spring.data.mongodb.port=27017
-#spring.data.mongodb.database=aas
-#spring.data.mongodb.username=admin
-#spring.data.mongodb.password=password
+#spring.data.mongodb.database=admin
```

Comment:
The admin database should be exclusive for mongo configuration -> e.g. authentication database.
The "business data" should be kept in business related databases - like "aas".
See: https://www.mongodb.com/docs/manual/tutorial/manage-users-and-roles/#create-a-user-defined-role